src/gui/gui.c

```diff
@@ -15,140 +15,139 @@
   // glfwWindowHint(GLFW_DECORATED, GLFW_FALSE);
   // glfwWindowHint(GLFW_FLOATING, GLFW_TRUE);
   qvk.window = glfwCreateWindow(qvk.win_width, qvk.win_height, "vkdt", NULL, NULL);
   glfwSetWindowPos(qvk.window, 0, 0);
 
-  if(!qvk.window)
+  if (!qvk.window)
   {
-    dt_log(s_log_gui|s_log_err, "could not create GLFW window");
+    dt_log(s_log_gui | s_log_err, "could not create GLFW window");
     return 1;
   }
 
   qvk.glfw_extensions = glfwGetRequiredInstanceExtensions(&qvk.num_glfw_extensions);
-  if(!qvk.glfw_extensions)
+  if (!qvk.glfw_extensions)
   {
-    dt_log(s_log_gui|s_log_err, "couldn't get GLFW instance extensions");
+    dt_log(s_log_gui | s_log_err, "couldn't get GLFW instance extensions");
     return 1;
   }
 
   dt_log(s_log_gui, "vk extension required by GLFW:");
-  for(int i = 0; i < qvk.num_glfw_extensions; i++)
+  for (int i = 0; i < qvk.num_glfw_extensions; i++)
     dt_log(s_log_gui, "  %s", qvk.glfw_extensions[i]);
 
-  if(qvk_init())
+  if (qvk_init())
   {
-    dt_log(s_log_err|s_log_gui, "init vulkan failed");
+    dt_log(s_log_err | s_log_gui, "init vulkan failed");
     return 1;
   }
 
   /* create surface */
-  if(glfwCreateWindowSurface(qvk.instance, qvk.window, NULL, &qvk.surface))
+  if (glfwCreateWindowSurface(qvk.instance, qvk.window, NULL, &qvk.surface))
   {
-    dt_log(s_log_qvk|s_log_err, "could not create surface!");
+    dt_log(s_log_qvk | s_log_err, "could not create surface!");
     return 1;
   }
 
   VkBool32 res;
   vkGetPhysicalDeviceSurfaceSupportKHR(qvk.physical_device, qvk.queue_idx_graphics, qvk.surface, &res);
   if (res != VK_TRUE)
   {
-    dt_log(s_log_qvk|s_log_err, "no WSI support on physical device");
+    dt_log(s_log_qvk | s_log_err, "no WSI support on physical device");
     return 1;
   }
 
   QVKR(dt_gui_recreate_swapchain());
 
   // create command pool and fences and semaphores.
   // assume image_count does not change, so we only need to do this once.
-  for(int i = 0; i < vkdt.image_count; i++)
+  for (int i = 0; i < vkdt.image_count; i++)
   {
     VkCommandPoolCreateInfo cmd_pool_create_info = {
-      .sType            = VK_STRUCTURE_TYPE_COMMAND_POOL_CREATE_INFO,
-      .queueFamilyIndex = qvk.queue_idx_graphics,
-      .flags            = VK_COMMAND_POOL_CREATE_RESET_COMMAND_BUFFER_BIT,
+        .sType = VK_STRUCTURE_TYPE_COMMAND_POOL_CREATE_INFO,
+        .queueFamilyIndex = qvk.queue_idx_graphics,
+        .flags = VK_COMMAND_POOL_CREATE_RESET_COMMAND_BUFFER_BIT,
     };
-    QVKR(vkCreateCommandPool(qvk.device, &cmd_pool_create_info, NULL, vkdt.command_pool+i));
+    QVKR(vkCreateCommandPool(qvk.device, &cmd_pool_create_info, NULL, vkdt.command_pool + i));
 
     VkCommandBufferAllocateInfo cmd_buf_alloc_info = {
-      .sType              = VK_STRUCTURE_TYPE_COMMAND_BUFFER_ALLOCATE_INFO,
-      .commandPool        = vkdt.command_pool[i],
-      .level              = VK_COMMAND_BUFFER_LEVEL_PRIMARY,
-      .commandBufferCount = 1,
```

Comment:
really? this is nonsense. it will just misalign the '='

i don't know why IDEs like to burn cpu cycles on this.